.github/workflows/build-and-test.yml

```diff
@@ -112,6 +112,28 @@ jobs:
           ~/actionutils.sh wait_for_osds 3
           sudo microceph.ceph -s
 
+      - name: Enable RGW
+        run: |
+          set -eux
+          juju config microceph enable-rgw=*
```

Comment:
Nit: as a precaution so that some shell doesn't try to expand this, maybe use
```
juju config microceph enable-rgw='*'
```

Reply:
Done